openedx/features/edly/cookies.py

```diff
@@ -42,10 +42,13 @@ def delete_logged_in_edly_cookies(response):
     Returns:
         HttpResponse
     """
-    response.delete_cookie(
+    response.set_cookie(
         settings.EDLY_USER_INFO_COOKIE_NAME,
         path='/',
-        domain=settings.SESSION_COOKIE_DOMAIN
+        domain=settings.SESSION_COOKIE_DOMAIN,
+        max_age=0,
+        expires='Thu, 01-Jan-1970 00:00:00 GMT',
+        secure=response._request.is_secure()  # pylint: disable=protected-access
```

Comment:
@burhandodhy I suggest we should not use private property of response object to get request instead we should do it like this
```
from crum import get_current_request
request = get_current_request()
```

Reply:
@ziafazal  done